universal-application-tool-0.0.1/test/controllers/admin/AdminQuestionControllerTest.java

```diff
@@ -267,6 +268,11 @@ public void update_setsIdsAsExpected() throws Exception {
     // We can only update draft questions, so save this in the DRAFT version.
     testQuestionBank.maybeSave(definition, LifecycleStage.DRAFT);
 
+    ArrayList<String> newOptions = new ArrayList<>();
```

Comment:
These were in the other merged PR right?

Reply:
:( should be fixed now